apps/docs/src/examples/inputs/TagsDropdown.tsx

```diff
@@ -0,0 +1,101 @@
+import { useEffect, useRef, useState } from "react";
+import {
+  HvBaseDropdown,
+  HvLabel,
+  HvPanel,
+  HvTag,
+} from "@hitachivantara/uikit-react-core";
+
+const colors = [
+  "Blue",
+  "Red",
+  "Green",
+  "Yellow",
+  "Purple",
+  "White",
+  "Black",
+  "Orange",
+  "Pink",
+  "Brown",
+  "Gray",
+  "Cyan",
+  "Magenta",
+  "Lime",
+  "Teal",
+  "Lavender",
+];
+
+export default function Demo() {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const focusTarget = useRef<HTMLDivElement>(null);
+
+  const [selectedColors, setSelectedColors] = useState<string[]>([]);
+
+  useEffect(() => {
+    if (containerRef.current) {
+      containerRef.current.scrollLeft = containerRef.current.scrollWidth;
+    }
+  }, [selectedColors]);
+
+  const handleAddColor = (color: string) => {
+    setSelectedColors((prev) => {
+      const newColors = prev.filter((c) => c !== color);
+      return newColors.length ? [...newColors, color] : [color];
+    });
+  };
+
+  const handleRemoveColor = (color: string) => {
+    setSelectedColors((prev) => prev.filter((c) => c !== color));
+  };
+
+  const focusOnContainer = () => {
+    focusTarget.current?.focus();
+  };
+
+  return (
+    <div className="w-300px">
+      <HvLabel>Tags dropdown input</HvLabel>
```

Comment:
If we're using `HvLabel`, then it needs to be accessible. (ie. `id`/`aria-labelledby`). Also, the `HvLabel`'s "label" is via the `label` prop, not `children`

Alternatively, we can use `aria-label` on the `HvBaseDropdown`